pkg/kudoctl/cmd/init_test.go

```diff
@@ -92,43 +94,46 @@ func TestInitCmd_output(t *testing.T) {
 	MockCRD(c.client, "certificates.cert-manager.io", "v1alpha2")
 	MockCRD(c.client, "issuers.cert-manager.io", "v1alpha2")
 
-	tests := []string{"yaml"}
+	tests := []output.Type{output.TypeYAML, output.TypeJSON}
```

Comment:
could we define all supported outputs maybe in the output.go file?

Reply:
Done